js/index.js

```diff
@@ -43,23 +47,46 @@ function renderPepperoni() {
 }
 
 function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(oneShroom => {
+    if (state.mushrooms) {
+      oneShroom.style.visibility = 'visible';
+    } else {
+      oneShroom.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(onePepper => {
+    if (state.greenPeppers) {
+      onePepper.style.visibility = 'visible';
+    } else {
+      onePepper.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  if (state.whiteSauce === true){
+    addWhiteSauce.className = "sauce"
```

Comment:
Algunos usaron classList y los metodos add y remove, no se los mostre en clase pero si mencione que podian usar classlist y la lógica suya 😄 
```suggestion
    addWhiteSauce.classLsit.add("sauce")
```